pkg/module/gitsource/git.go

```diff
@@ -53,8 +47,8 @@ func (g GitSource) FetchSource(ctx cpi.Context, path, repo, version, gitRemote s
 	}, nil
 }
 
-func (g GitSource) determineRepositoryURL(gitRemote string) (string, error) {
-	r, err := git.PlainOpen(".")
+func (g GitSource) DetermineRepositoryURL(gitRemote string, repoPath string) (string, error) {
```

Comment:
btw, I covered this test in e2e test setup https://github.com/kyma-project/cli/pull/1768/files#r1329956985